Scripts/get_version.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/python3
```

Comment:
What context to you plan to use this? I know we talked about some helpers for the Prepare-Release scripts and I think we can use this there but I'm curious if you have other plans for it as well?

Reply:
I had written this before then as preparation for a legitimate release stage in our pipeline to update the git tag for the release. I wanted to do that with a bash script (which I loathe) but trying to parse the results of `git describe` proved so frustrating and brittle that I moved to python. The idea here is that in my bash script, I can now simply say:
```bash
version=python3 version.py current
# do other bash-y things here
```

Reply:
Cool before you go and write your own step of scripts/steps that do the git tagging please see my comment in your other PR as I'd like to use our common scripts as much as possible. We will need to light up some common infrastructure for both the tagging, github.io common publishing, and prepare-release. So the more we light up the more common infra you guys can leverage.